bin/all_sky_search/pycbc_fit_sngls_split_binned

```diff
@@ -110,60 +111,46 @@ trigf = h5py.File(args.trigger_file, 'r')
 
 logging.info('Opening template file: %s' % args.bank_file)
 bank = h5py.File(args.bank_file, 'r')
-logging.info('setting up template bank parameters')
-params = {
-        "mass1" : bank['mass1'][:],
-        "mass2" : bank['mass2'][:],
-        "spin1z": bank['spin1z'][:],
-        "spin2z": bank['spin2z'][:]
-}
-bank.close()
+logging.info('Getting template bank parameters')
 
-# Only calculate params if needed
+# Calculate params needed
 usedparams = [args.split_param_two, args.split_param_one, args.bin_param]
-extparams = [par for par in ['mchirp', 'eta', 'chi_eff', 'template_duration']
-             if par in usedparams]
 
-for ex_p in extparams:
+params = {}
+for ex_p in usedparams:
```

Comment:
much simpler now - just rename the variable to something less obscure, eg `for par in usedparams`